edge/client/portainer_edge_async_client.go

```diff
@@ -147,6 +167,36 @@ func (client *PortainerAsyncClient) GetEnvironmentStatus(flags ...string) (*Poll
 				}
 			}
 
+			for _, stack := range client.stackLogCollectionQueue {
+				cs, err := docker.GetContainersWithLabel(fmt.Sprintf("com.docker.compose.project=edge_%s", stack.EdgeStackName))
```

Comment:
this will miss standalone

	ComposeStackNameLabel = "com.docker.compose.project"
	SwarmStackNameLabel   = "com.docker.stack.namespace"

Reply:
Hmm, what do you mean? I tested this using edge stacks on Docker Standalone and it worked.